src/components/styled-atlaskit/Select/Select.tsx

```diff
@@ -5,22 +5,30 @@
  */
 
 import React from "react";
-import { default as AkSelect, SelectProps as AkSelectProps } from "@atlaskit/select";
+import {
+  default as AkSelect,
+  SelectProps as AkSelectProps,
+  CreatableSelect,
+  OptionType,
+  OptionsType,
+} from "@atlaskit/select";
 import { SelectOption } from "../../../types";
 import "./Select.scss";
 
 export interface SelectProps extends Omit<AkSelectProps<SelectOption, boolean>, "onChange"> {
   onChange?: (option: SelectOption) => void;
+  isCreatable?: boolean;
 }
 
-export const Select = ({ isMulti = false, onChange, ...props }: SelectProps) => {
-  return (
-    <AkSelect
-      className="styled-atlaskit-select"
-      classNamePrefix={"styled-atlaskit-select"}
-      isMulti={isMulti}
-      onChange={onChange ? (item) => onChange(item as SelectOption) : undefined}
-      {...props}
-    />
-  );
+export const Select = ({ isMulti = false, isCreatable = false, onChange, ...props }: SelectProps) => {
+  const defaultProps = {
+    isMulti,
+    className: "styled-atlaskit-select",
+    classNamePrefix: "styled-atlaskit-select",
+    onChange: onChange
+      ? (item: OptionType | OptionsType<OptionType> | null) => onChange(item as SelectOption)
```

Comment:
why do we add the type for `item`?

Reply:
If the prop is called directly like this `<Select onChange={(item) => onChange(item as SelectOption)}>`, the type is inferred from the Select component so no need to add type, but not when defined from constant. 

If not defined we will get warning `Parameter 'item' implicitly has an 'any' type`.